src/Apps/Artwork/Components/PrivateArtwork/PrivateArtworkAboutArtist.tsx

```diff
@@ -22,6 +22,10 @@ export const PrivateArtworkAboutArtist: React.FC<PrivateArtworkAboutArtistProps>
     graphql`
       fragment PrivateArtworkAboutArtist_artwork on Artwork {
         title
+        artist {
+          name
+          blurb
+        }
```

Comment:
Should this still be "on Artwork" on line 23 if I'm trying to access artist information? @damassi